bin/duo-install

```diff
@@ -0,0 +1,339 @@
+#!/usr/bin/env node
+
+/**
+ * Module dependencies.
+ */
+
```

Comment:
looks like a lot of this is copypasta from `bin/duo_`.  is there any way we could share some code here?